### DISCUSSION TITLE
i2C on ILI9486

### DISCUSSION THREAD

**Discussion by Rhythm8503** [Bodmer/TFT_eSPI#3022] [Alerts]
Hello! I'm using an ESP32 WROOM Devkit with the Arduino IDE, for some reason when using i2C with the display it causes Core 1 to panic, I'm currently using the ILI9486 in 8-bit parallel but I don't understand how this would conflict with i2C writing.

**Comment by Bodmer** [comment]
It maybe the watchdog timing out. Try adding yield() in any long duration functions.

Another possibility is that cross talk from the high speed TFT GPIO  is coupling into the higher impedance I2C lines. Try to keep the physically separated.
